### DISCUSSION TITLE
Filtering

### DISCUSSION THREAD

**Discussion by csirikk** [the-marenga/sf-scrapbook-helper#40] [Ideas]
Filtering attackers as it is done now, by level, does not seem optimal. A better approach might be the total attribute number. The level isn't the most reliable way to tell if you can win a fight, while the total attribute number surely is (at least without RNG).

**Comment by csirikk** [comment]
I understand levels might be set as the default filter since many items appear at specific ranges, yet if you really dont want to lose fights and miss out on a free attack a second filter would be ideal.

**Comment by the-marenga** [comment]
Sort/Filter by attributes like a good idea
